common/taskana-common-data/src/main/resources/sql/sample-data/workbasket.sql

```diff
@@ -1,12 +1,12 @@
  -- KSC workbaskets
 -- WORKBASKET TABLE           (ID                                        , KEY          , CREATED              , MODIFIED             , NAME                       , DOMAIN    , TYPE      , DESCRIPTION                  , OWNER        , CUSTOM_1 , CUSTOM_2  , CUSTOM_3 , CUSTOM_4  , ORG_LEVEL_1   , ORG_LEVEL_2, ORG_LEVEL_3, ORG_LEVEL_4, MARKED_FOR_DELETION  );
```

Comment:
Please add/remove spaces so that after your changes all lines are aligned to this header.